mlflow/utils/requirements_utils.py

```diff
@@ -260,9 +266,15 @@ def _capture_imported_modules(model_uri, flavor):
 
     local_model_path = _download_artifact_from_uri(model_uri)
 
+    process_timeout = os.environ.get("REQUIREMENTS_INFERENCE_TIMEOUT", 2 * 60)
```

Comment:
Can we add this to `mlflow.environment_variables` with docs and also prefix it with `MLFLOW_`?

Reply:
definitely!